Sources/DictionaryExtensions.swift

```diff
@@ -8,10 +8,10 @@
 import UIKit
 
 extension Dictionary {
+   
     /// EZSE: Returns the value of a random Key-Value pair from the Dictionary
-    public func random() -> Value {
-        let index: Int = Int(arc4random_uniform(UInt32(self.count)))
-        return Array(self.values)[index]
+    public func random() -> Value? {
+        return Array(self.values).random()
```

Comment:
Yes I believe so; I generally only spot common `self` usage in the stdlib's source code when subscripting (in which case in naturally must be present; `self[subscript]`), but at the same time it seems they (stdlib) don't follow this implicitness extremely strictly (I've seen both, but implicits are more frequent).